### PR TITLE
[Core] Removing old precated  and not used SuperLU solver

### PR DESCRIPTION
**Pull request by loumalouomega**
Is not even included in any file, and not exported to python

### CODE REVIEW

File: kratos/linear_solvers/superlu_solver.h [hunk 1]
```diff
@@ -1,288 +0,0 @@
...
-* =======================================================================*
-* kkkk   kkkk  kkkkkkkkkk   kkkkk    kkkkkkkkkk kkkkkkkkkk kkkkkkkkkK    *
-* kkkk  kkkk   kkkk   kkkk  kkkkkk   kkkkkkkkkk kkkkkkkkkk kkkkkkkkkK    *
-* kkkkkkkkk    kkkk   kkkk  kkkkkkk     kkkk    kkk    kkk  kkkk         *
-* kkkkkkkkk    kkkkkkkkkkk  kkkk kkk	kkkk    kkk    kkk    kkkk       *
-* kkkk  kkkk   kkkk  kkkk   kkkk kkkk   kkkk    kkk    kkk      kkkk     *
-* kkkk   kkkk  kkkk   kkkk  kkkk  kkkk  kkkk    kkkkkkkkkk  kkkkkkkkkk   *
-* kkkk    kkkk kkkk    kkkk kkkk   kkkk kkkk    kkkkkkkkkk  kkkkkkkkkk 	 *
-*                                                                        *
-* krATos: a fREe opEN sOURce CoDE for mULti-pHysIC aDaptIVe SoLVErS,     *
```
Comment (loumalouomega) [inline kratos/linear_solvers/superlu_solver.h:22]: This header is so messenger, missing early 2000's :P

**Review by roigcarlo — COMMENTED** [review]
Cool to me, but I would like @RiccardoRossi approval as well

**Comment by loumalouomega** [comment]
Ping @RiccardoRossi

File: kratos/linear_solvers/superlu_solver.h (hunk 1 above)
Comment (philbucher) [reply kratos/linear_solvers/superlu_solver.h:22]: 2tehot

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**